docs/guides/testing.md

```diff
@@ -109,3 +109,18 @@ sudo /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=1024
 sudo /sbin/mkswap /var/swap.1
 sudo /sbin/swapon /var/swap.1
 ```
+
+## Static Analysis
+
+Version 2.x is also covered by static analysis courtesy of [PHPStan](https://github.com/phpstan/phpstan). This analyzes our code files and structure to ensure things like existence and accessibility of called methods and functions, number of passed arguments, data types in docs, etc.
+
+To run, use:
+
+```
+vendor/bin/phpstan analyze -l 1
+```
+
+Where the last argument is the level of strictness (0-4) to apply. We're currently starting at level 1 and working our way up over time.
```

Comment:
I think there are 8 levels, right? Or does this mean that we only intend to reach level 4 for the moment (which I think would also be reasonable).

Reply:
good call, I copied that from the initial release posting and not the newest levels of support

Reply:
Cool, so 7 it is, good to know 😄